packages/web-frontend/src/components/View/MatrixWrapper/MatrixWrapper.js

```diff
@@ -163,6 +163,7 @@ export class MatrixWrapper extends Component {
     return (
       !shallowEqual(this.props, nextProps) ||
       nextState.isLoading !== this.state.isLoading ||
+      nextState.expandedMatrixData !== this.state.expandedMatrixData ||
```

Comment:
Not quite sure about why this is needed now?

Reply:
hmmm me neither. I guess react is not picking up that the props have changed in EnlargedDialog when the useEffect kicks in. Perhaps because viewContent is the object that gets passed in to EnlargedDialogContent and it is a nested object (some shallow comparison issue). Anyway if this works then I think it is ok.

Reply:
Awesome, thanks.